packages/puppeteer-core/src/bidi/Browser.ts

```diff
@@ -106,6 +106,12 @@ export class BidiBrowser extends Browser {
       }) as [string, ...string[]],
     );
 
+    await session.send('network.addDataCollector', {
```

Comment:
you need to catch and suppress the error here because Firefox does not support this yet